actionview/lib/action_view/template/handlers/erb.rb

```diff
@@ -58,6 +58,7 @@ def add_expr_escaped(src, code)
         def add_stmt(src, code)
           flush_newline_if_pending(src)
           super
+          flush_newline_if_comment(src, code)
```

Comment:
I'm a away for a couple weeks and don't have my laptop. Will get back to it when I'm coming back. Thanks for your comment!